bundles/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/QueuedSend.java

```diff
@@ -48,7 +49,7 @@ public class QueuedSend implements Runnable, Closeable {
      */
     public void start() {
         willbeclosed = false;
-        thread = new Thread(this);
+        thread = new Thread(this, "OH-binding-" + MilightBindingConstants.BINDING_ID + "-QueueSend");
```

Comment:
Should it match class name?
```suggestion
        thread = new Thread(this, "OH-binding-" + MilightBindingConstants.BINDING_ID + "-QueuedSend");
```

Reply:
IMHO it is better to describe the function rather than the class name.

Reply:
But if you are just thinking of the spelling, I am fine with that.